iidm/iidm-geodata/src/main/java/com/powsybl/iidm/geodata/odre/OdreGeoDataAdderPostProcessor.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.geodata.odre;
+
+import com.google.auto.service.AutoService;
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.commons.config.PlatformConfig;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.iidm.network.ImportPostProcessor;
+import com.powsybl.iidm.network.Network;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * @author Hugo Kulesza {@literal <hugo.kulesza at rte-france.com>}
+ */
+@AutoService(ImportPostProcessor.class)
+public class OdreGeoDataAdderPostProcessor implements ImportPostProcessor {
+
+    public static final String NAME = "odreGeoDataImporter";
+    public static final Map<String, String> DEFAULT_FILE_NAMES = Map.of("substations", "postes-electriques-rte.csv",
+            "aerial-lines", "lignes-aeriennes-rte-nv.csv",
+            "underground-lines", "lignes-souterraines-rte-nv.csv");
```

Comment:
It would be cleaner to change the default filenames as `substations.csv`, `aerial-lines.csv` and `underground-lines.csv` and to rename the test files accordingly.

In addition, the default file names should be described in the documentation.